code/datums/gamemodes/spy_theft.dm

```diff
@@ -397,6 +397,7 @@
 	station_bounties[/obj/item/pen/crayon/golden] = 2
 	station_bounties[/obj/item/remote/porter/port_a_sci] = 2
 	station_bounties[/obj/item/clothing/suit/hosmedal] = 3
+	station_bounties[obj/item/manudrive/eng] = 2
```

Comment:
Linter is angry about the / missing in front

Reply:
Ah fuck me thought I fixed that, was tired when I made this pr so its not suprising theres some sloppy mistakes.